backend/src/main/java/com/google/step/data/Form.java

```diff
@@ -17,77 +17,84 @@
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.Key;
 import com.google.appengine.api.datastore.KeyFactory;
-
+import com.google.appengine.api.users.User;
+import com.google.step.utils.AdvertiserUtil;
 import java.util.Date;
 
 /**
- * This class represents a Form and all of its data.
- * Supports conversion to datastore Entity objects and back.
+ * This class represents a Form and all of its data. Supports conversion to datastore Entity objects
```

Comment:
nit: remove "This class" and just do "Represents". Generally, starting with a verb is preferred.

Reply:
Fixed. planning to do a quick look over all the javadocs soon.